src/config/2019.json

```diff
@@ -325,6 +325,15 @@
       "github": "andydavies",
       "twitter": "AndyDavies"
     },
+    "arigaud-ca": {
+      "name": "Arthur Rigaud",
+      "teams": [
+        "translators"
+      ],
+      "avatar_url": "https://avatars2.githubusercontent.com/u/50932885?v=4&s=200",
+      "github": "arigaud-ca",
+      "twitter": "arigaud_ca"
+    },
```

Comment:
@arigaud-ca can you confirm you're happy with this info? and or change it as you see fit.